planning/ThingDescription/td-next-work-items/usability-and-design.md

```diff
@@ -72,32 +72,92 @@ Related Issues:
   - The initial connection must not be mandatory to establish upon TD consumption and should left to the implementation when to establish and close it.
 
 **Notes:**
+
 - When to close the connection needs to be discussed.
 - Complex security mechanisms exchange should be handled at the same time
 
-#### Examples of Message Flow
+#### Examples of Message Sequences
+
+##### Simple HTTP Connection
+
+- Participating Entities
+
+![Participating Entities](./images/initial-connection-HTTP-entities.png)
+
+- Lifecycle of a Connection
+
+![Lifecycle of a Connection](./images/initial-connection-HTTP-lifecycle.png)
+
+- Message Sequence
+
+![Message Sequence](./images/initial-connection-HTTP-sequence.png)
+
+##### No security Broker Connections
+
+- Participating Entities
+
+![Participating Entities](./images/initial-connection-Broker-entities.png)
+
+- Lifecycle of a Connection
+
+![Lifecycle of a Connection](./images/initial-connection-Broker-lifecycle.png)
+
+- Message Sequence
+
+![Message Sequence](./images/initial-connection-Broker-sequence.png)
+
+##### Basic WebSocket Connections
 
-- No security Broker Connections
+- Participating Entities
 
-> Example message flow here
+![Participating Entities](./images/initial-connection-WS-entities.png)
 
-- WebSocket with Subprotocol Connection
+- Lifecycle of a Connection
+
+TODO
+
+- Message Sequence
+
+![Message Sequence](./images/initial-connection-WS-sequence.png)
+
+##### WebSocket with Subprotocol Connection
 
 > Example message flow from WebThings here
 
 > Example message flow from Socket.io here
 
 > Example message flow from CoAP over WS here (see https://www.rfc-editor.org/rfc/rfc8323.html)
 
-- Initial Token/Credential exchange
+##### OAuth2-based Interaction
+
+- Participating Entities
+
+![Participating Entities](./images/initial-connection-Oauth2-entities.png)
+
+- Lifecycle of a Connection
```

Comment:
@relu91 to change it to Lifecycle of the Session. The token is associated to the session of course